backblaze/store_test.go

```diff
@@ -0,0 +1,136 @@
+package backblaze_test
+
+import (
+	"bufio"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/araddon/gou"
+	"github.com/bmizerany/assert"
+	"github.com/lytics/cloudstorage"
+	"github.com/lytics/cloudstorage/backblaze"
+)
+
+var config = &cloudstorage.Config{
+	Type:       backblaze.StoreType,
+	AuthMethod: backblaze.AuthKey,
+	Bucket:     os.Getenv("BACKBLAZE_BUCKET"),
+	TmpDir:     "/tmp/localcache/backblaze",
+	Settings:   make(gou.JsonHelper),
+}
+
+func TestConfig(t *testing.T) {
+
+	if config.Bucket == "" {
+		t.Logf("must provide BACKBLAZE_BUCKET, BACKBLAZE_ACCESS,  BACKBLAZE_KEY  env vars")
```

Comment:
This comment seems wrong?   Should be `BACKBLAZE_ACCOUNT` ?  not access?

Reply:
Your're right. I fixed it.